Documentation/RELEASE_NOTES.md

```diff
@@ -29,6 +29,10 @@
 * Allow second disk to be mounted to VSTAT (ES)
 * Add support for emailing health reports (METROAE-1332)
 * Add support for symlinks as files
+<<<<<<< HEAD
```

Comment:
These lines beginning with `<<<<<<<`, `>>>>>>` and `======` are comment lines added by git when there was a merge conflict. Please clean up this file. These lines should not be present.